### DISCUSSION TITLE
What is the future for Pages Router?

### DISCUSSION THREAD

**Discussion by MarcusCaspeco** [vercel/next.js#61575] [App Router] [answered]
Is there an official statement regarding the future of the Pages Router?

The docs says this:
> The Pages Router is the original Next.js router, which allowed you to build server-rendered React applications and **continues to be supported for older Next.js applications**. 

This, to me, hints at a future deprecation notice. But is there an official statement or plan to deprecate it soon, and if so, is there a timeline?

It feels like developers are left in sort of a limbo right now: the App Router is not supported by many third party packages and the Pages Router may be deprecated soon. It would be great to get some clarification about the plans for Routing in Next.js in the coming years, so we can prioritize properly.

**Comment by erkesado** [comment]
I'm also interested to know it

**Comment by renbrek** [comment, marked as answer]
https://github.com/vercel/next.js/discussions/56655#discussion-5721236

**Comment by ahmedmusawir** [comment]
Yes, I'm interested also ... cuz the app router sux!
